### PR TITLE
[GPII-3566]: Enable Stackdriver Slack notifications for stg, prd

### PR DESCRIPTION
**Pull request by natarajaya**
I authorized Stackdriver to send notifications to RTF Slack *#alerts* channel for `gpii-gcp-stg` and `gpii-gcp-prd` projects (see instructions in [updated README](https://github.com/natarajaya/gpii-infra/blob/5fa2511d3e639e970a354101d2c8f8cbda4cbf24/gcp/README.md#to-receive-notifications-in-slack)).
This adds Slack notification channel config and changes to Stackdriver Ruby client to work properly with Slack notification channels.

### CODE REVIEW

**Comment by stepanstipl** [comment]
lgtm

File: shared/rakefiles/stackdriver.rb [hunk 1]
```diff
@@ -131,14 +133,22 @@ def process_notification_channels(notification_channels = [])
 
     if stackdriver_notification_channels[notification_channel_identifier]
       notification_channel["name"] = stackdriver_notification_channels[notification_channel_identifier]["name"]
-      puts "Updating notification channel \"#{notification_channel_identifier}\"..."
-      notification_channel_service_client.update_notification_channel(notification_channel)
+      unless notification_channel["immutable"]
+        puts "Updating notification channel \"#{notification_channel_identifier}\"..."
+        notification_channel_service_client.update_notification_channel(notification_channel)
+      else
+        puts "Skipping update of notification channel \"#{notification_channel_identifier}\"..."
```
Comment (mrtyler) [inline shared/rakefiles/stackdriver.rb]: How about "immutable notification channel"?

File: shared/rakefiles/stackdriver.rb [hunk 2]
```diff
@@ -131,14 +133,22 @@ def process_notification_channels(notification_channels = [])
...
+      end
     else
-      puts "Creating notification channel \"#{notification_channel_identifier}\"..."
-      notification_channel = notification_channel_service_client.create_notification_channel(formatted_parent, notification_channel)
+      unless notification_channel["immutable"]
+        puts "Creating notification channel \"#{notification_channel_identifier}\"..."
+        notification_channel = notification_channel_service_client.create_notification_channel(formatted_parent, notification_channel)
+      else
+        puts "Skipping creation of notification channel \"#{notification_channel_identifier}\"..."
```
Comment (mrtyler) [inline shared/rakefiles/stackdriver.rb]: How about "immutable notification channel"?

File: shared/rakefiles/stackdriver.rb (hunk 1 above)
Comment (natarajaya) [reply shared/rakefiles/stackdriver.rb]: Makes sense.

File: shared/rakefiles/stackdriver.rb (hunk 2 above)
Comment (natarajaya) [reply shared/rakefiles/stackdriver.rb]: Done.

**Pull request merged**